lanuv_download/resources/cron/tab.txt

```diff
@@ -1,3 +1,4 @@
 SHELL=/bin/bash
 PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
-40 * * * * /bin/bash /scripts/dl_lanuv.sh >> /log.txt 2>&1
\ No newline at end of file
+40 * * * * /bin/bash /scripts/dl_lanuv.sh >> /log.txt 2>&1
+* 9,21 * * * /bin/bash /scripts/dl_lanuv_no.sh >> /log.txt 2>&1
```

Comment:
This means execution at 9am and 9pm, right? maybe a comment in the file would be good for the future.